doc/user/release-notes.html

```diff
@@ -56,6 +56,8 @@ <h2>Release notes for Crux</h2>
 
 <h4>Minor changes</h4>
 <ul>
+  <li>01 Apr 2022: XCorr scoring implemented in C++, temporarily solved bug in threading issue: https://github.com/crux-toolkit/crux-toolkit/issues/407 
+  <li>01 Apr 2022: Fixed bugs in the speed tide update. Updated smoke-tests, timing tests. </li> 
```

Comment:
No need to mention that the smoke tests and timing tests were updated.
Also, I don't think there is any need to talk about fixing these bugs, since the fixes only happened on this branch, right? They were never released.
The date for each of these updates should be the date when the change is merged into the trunk.